From 06dc32a9f3ede351c46167afa5002a0fe062c0f7 Mon Sep 17 00:00:00 2001
From: Steve Beattie <gitlab@nxnw.org>
Date: Thu, 26 Aug 2021 17:43:43 +0000
Subject: [PATCH] utils: Add new python versions to logprof.conf

Adding everything up to 3.19 should make the file future-proof for a
while ;-)

Fixes: https://gitlab.com/apparmor/apparmor/-/issues/193
MR: https://gitlab.com/apparmor/apparmor/-/merge_requests/795
Acked-by: Steve Beattie <steve.beattie@canonical.com>
(cherry picked from commit 4559a2997cf162b0f54f602180fd352e8d2486c1)
Signed-off-by: Steve Beattie <steve.beattie@canonical.com>
---
 utils/logprof.conf | 12 ++++++++++++
 1 file changed, 12 insertions(+)

--- a/utils/logprof.conf
+++ b/utils/logprof.conf
@@ -112,6 +112,18 @@
   /usr/bin/python3.5    = icn
   /usr/bin/python3.6    = icn
   /usr/bin/python3.7    = icn
+  /usr/bin/python3.8    = icn
+  /usr/bin/python3.9    = icn
+  /usr/bin/python3.10   = icn
+  /usr/bin/python3.11   = icn
+  /usr/bin/python3.12   = icn
+  /usr/bin/python3.13   = icn
+  /usr/bin/python3.14   = icn
+  /usr/bin/python3.15   = icn
+  /usr/bin/python3.16   = icn
+  /usr/bin/python3.17   = icn
+  /usr/bin/python3.18   = icn
+  /usr/bin/python3.19   = icn
   /usr/bin/tr   = icn
 
 [required_hats]
